pychunkedgraph/graph/meta.py

```diff
@@ -80,7 +84,20 @@ def custom_data(self):
     def ws_cv(self):
         if self._ws_cv:
             return self._ws_cv
-        self._ws_cv = CloudVolume(self._data_source.WATERSHED)
+
+        try:
+            # try reading a cached info file for distributed workers
+            # useful to avoid md5 errors on high gcs load
+            cache_key = f"{self.graph_config.ID}:cv_info_cached"
```

Comment:
Define this before the `try` statement. I would add "watershed" to the key to make this unambiguous

Reply:
The cache key is only used within the try block, no need to define it outside. Will add `watershed`.